platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -59,15 +63,15 @@ public ExportCommand(ConfigurationMigrationService configurationMigrationService
         this(configurationMigrationService, security, pathBuilder.build());
     }
 
-    ExportCommand(ConfigurationMigrationService configurationMigrationService,
-            Security security, Path path) {
+    ExportCommand(ConfigurationMigrationService configurationMigrationService, Security security,
```

Comment:
I was thinking something like [this](https://github.com/apache/karaf/blob/karaf-4.0.7/config/src/test/java/org/apache/karaf/config/command/EditCommandTest.java#L50). No constructor arguments anywhere. Only setters for `@Reference` variables.